stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkEventReporterImpl.java

```diff
@@ -39,6 +48,8 @@
 
   private static NetworkEventReporter sInstance;
 
+  private static final CharsetDecoder decoder = Charset.forName(Utf8Charset.NAME).newDecoder();
```

Comment:
Make this a non-static member variable and initialize it in the constructor.  No need to muck up the `get()` method below.